File: Client/Projekcija.js
export class Projekcija{
    
    constructor(id, datum, salaid, filmid){
        this.id = id;
        this.datum = datum;
        this.salaid = salaid;
        this.filmid = filmid;
        this.rezervacije = [];
    }

    DodajRezervaciju(rezervacija){
        if(this.id === rezervacija.projekcija.id){
            this.rezervacije.push(rezervacija);
        }
    }
}

File: Models/Projekcija.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text.Json.Serialization;
using System;

namespace Models
{
    public class Projekcija
    {
        [Key]
        public int ID { get; set; }
       
        [Required]
        public DateTime Datum{ get; set; }

        public Sala Sala { get; set; }

        public List<Rezervacija> Rezervacije { get; set; }

        public Film Film { get; set; }
    }
}